### DISCUSSION TITLE
Converting content saved as JSON to HTML before display

### DISCUSSION THREAD

**Discussion by scvn1** [awcodes/filament-tiptap-editor#118] [Q&A]
I need to read the content programmatically in some cases, so I have saved the blog to the DB as JSON. However, when I am displaying that content on a regular HTML page in a Blade template, I need to output that to HTML.

How do I convert the TipTap JSON to HTML when I need to?

**Comment by awcodes** [comment]
This is something I'm currently working on. Stay tuned.

**Comment by awcodes** [comment]
Available in latest release.

```blade
{!! tiptap_converter()->asHTML($post->content) !!}
{!! tiptap_converter()->asJSON($post->content) !!}
{!! tiptap_converter()->asText($post->content) !!}
```

**Reply by scvn1** [reply]
Awesome! Thanks for this! Will try it out.
